image-layout.md

```diff
@@ -162,7 +162,7 @@ Those tags will often be represented in an image-layout repository with matching
 
 ### Index Example
 
-```json,title=Image%20Index&mediatype=application/vnd.oci.image.index.v1%2Bjson
+```json
```

Comment:
I think we still want to validate this example as an index even if it doesn't occur in `image-index.md`.  And titles seem like they'd always be a good idea.  Why are you removing this?